cdcsdk-testing/src/test/java/com/yugabyte/cdcsdk/testing/PostgresConsumerIT.java

```diff
@@ -0,0 +1,141 @@
+package com.yugabyte.cdcsdk.testing;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.ResultSet;
+import java.sql.Statement;
+import java.util.*;
```

Comment:
Importing everything with `*` is generally not considered a good practice if you only plan to use one or two classes.